PeerioTechnologies/peerio-mobile
# Feat telemetry new signup

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/10477/mobile-mixpanel-sign-up-trackers
https://app.clubhouse.io/peerio/story/13348/mobile-mixpanel-tracker-bugs
https://app.clubhouse.io/peerio/story/13302/mobile-update-text-input-helper-error-messages-for-sign-up-in
#### Testing instructions  
Verify that all telemetry events specified in the confluence docs (https://peerio.atlassian.net/wiki/spaces/DA/pages/206602280/Diff+Sign+In+Up+Trackers+Mobile) are send correctly with the correct parameters.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
